ee/clickhouse/queries/cohort_query.py

```diff
@@ -323,7 +326,7 @@ def _get_condition_for_property(self, prop: Property, prepend: str, idx: int) ->
                 res, params = self.get_performed_event_regularly(prop, prepend, idx)
         elif prop.type == "person":
             res, params = self.get_person_condition(prop, prepend, idx)
-        elif prop.type == "cohort":
+        elif prop.type in ["cohort", "static-cohort", "precalculated-cohort"]:
```

Comment:
https://github.com/PostHog/posthog/pull/9704 should handle "cohort" and "precalculated-cohort" by just unwrapping the properties so they're directly handled